webpack.config.ts

```diff
@@ -449,6 +457,7 @@ const appConfig: Configuration = {
 
     modules: ['node_modules'],
     extensions: ['.jsx', '.js', '.json', '.ts', '.tsx', '.less'],
+    symlinks: false,
```

Comment:
q: Whats the reason for this change?

Reply:
Referencing Webpack documentation for improving the resolve speed: https://webpack.js.org/guides/build-performance/#resolving

```
Resolving
The following steps can increase resolving speed:

...
- Set resolve.symlinks: false if you don't use symlinks (e.g. npm link or yarn link).
...
```